test/rpc-test.js

```diff
@@ -0,0 +1,227 @@
+  });
+
+  it('should rpc selectwallet', async () => {
+    const response = await wclient.execute('selectwallet', ['miner']);
+    assert.strictEqual(response, null);
+  });
+
+  it('should rpc getnewaddress from default account', async () => {
+    const acctAddr = await wclient.execute('getnewaddress', []);
+    assert(acctAddr);
+  });
+
+  it('should fail rpc getnewaddress from nonexistent account', async () => {
+    await assert.asyncThrows( async () =>
+      await wclient.execute('getnewaddress', ['bad-account-name']),
+      'Account not found.');
+  });
+
+  it('should rpc getaccountaddress', async () => {
+    addressMiner = await wclient.execute('getaccountaddress', ['default']);
+    assert(addressMiner);
+  });
+
+  it('should rpc generatetoaddress', async () => {
+    blocks = await nclient.execute('generatetoaddress',
+      [10, addressMiner]);
+    assert.strictEqual(blocks.length, 10);
+  });
+
+  it('should rpc sendtoaddress', async () => {
+    const acctHotDefault = await walletHot.getAccount('default');
+    addressHot = acctHotDefault.receiveAddress;
+
+    txid = await wclient.execute('sendtoaddress', [addressHot, 0.1234]);
+    assert.strictEqual(txid.length, 64);
+  });
+
+  it('should rpc sendmany', async () => {
+    const sendTo = {};
+    sendTo[addressHot] = 1.0;
+    sendTo[addressMiner] = 0.1111;
+    txid = await wclient.execute('sendmany', ['default', sendTo]);
+    assert.strictEqual(txid.length, 64);
+  });
+
+  it('should fail malformed rpc sendmany', async () => {
+    await assert.asyncThrows( async () =>
+      await wclient.execute('sendmany', ['default', null]),
+      'Invalid send-to address'
+    );
+
+    const sendTo = {};
+    sendTo[addressHot] = null;
+    await assert.asyncThrows( async () =>
+      await wclient.execute('sendmany', ['default', sendTo]),
+      'Invalid amount.'
+    );
+  });
+
+  it('should rpc listreceivedbyaddress', async () => {
+    await wclient.execute('selectwallet', ['hot']);
+
+    const listZeroConf = await wclient.execute('listreceivedbyaddress',
+      [0, false, false]);
+    assert.deepStrictEqual(listZeroConf, [{
+      'involvesWatchonly': false,
+      'address': addressHot,
+      'account': 'default',
+      'amount': 1.1234,
+      'confirmations': 0,
+      'label': ''
+    }]);
+
+    blocks.push(await nclient.execute('generatetoaddress', [1, addressMiner]));
+    await wdb.syncChain();
+
+    const listSomeConf = await wclient.execute('listreceivedbyaddress',
+      [1, false, false]);
+    assert.deepStrictEqual(listSomeConf, [{
+      'involvesWatchonly': false,
+      'address': addressHot,
+      'account': 'default',
+      'amount': 1.1234,
+      'confirmations': 1,
+      'label': ''
+    }]);
+
+    const listTooManyConf = await wclient.execute('listreceivedbyaddress',
+      [100, false, false]);
+    assert.deepStrictEqual(listTooManyConf, []);
+  });
+
+  it('should rpc listtransactions', async () => {
```

Comment:
You may also be able to add a test case here were the name is sent, as was fixed.